submission-scripts/script-lsf-mpirun/magpie.lsf-mpirun

```diff
@@ -2155,6 +2158,149 @@ export ZOOKEEPER_LOCAL_DIR="/tmp/${USER}/zookeeper"
 #
 # export ZOOKEEPER_SYNCLIMIT=5
 
+############################################################################
+# Alluxio Core Configurations
+############################################################################
+
+# Should Alluxio be run
+#
+# Specify yes or no. Defaults to no.
+#
+export ALLUXIO_SETUP=no
+
+# Alluxio Version
+#
+export ALLUXIO_VERSION="2.3.0"
+
+# Path to your Alluxio build/binaries
+#
+# This should be accessible on all nodes in your allocation. Typically
+# this is in an NFS mount.
+#
+export ALLUXIO_HOME="${HOME}/alluxio-${ALLUXIO_VERSION}"
+
+# ALLUXIO_WORKER_ON_MASTER
+#
+# Most big data projects do not have a "slave" or "worker" that also
+# runs on the master node (such as Hadoop, Hbase, etc.). Alluxio is
+# an exception in which it often likes to have a worker running on the
+# master node for convenience of running tests and such.
+#
+# It is not required to run a worker on the master, but it can be
+# convenient. Specify the below to "yes" to run a worker on the
+# master.
+#
+export ALLUXIO_WORKER_ON_MASTER="yes"
+
+# Path to store data local to each cluster node, typically something
+# in /tmp. This will store local conf files and log files for your
+# job. If local scratch space is not available, consider using the
+# MAGPIE_NO_LOCAL_DIR_DIR option. See README for more details.
+#
+export ALLUXIO_LOCAL_DIR="/tmp/${USER}/alluxio"
+
+# Path to store Alluxio Under File System data
+#
+# This should point to the path where you store your data
+# and have to be accessible on all nodes in your allocation.
+# It could be NFS mounted on all Alluxio nodes,
+# like "${ALLUXIO_HOME}/underFSStorage",
+# or ex. HDFS, like "hdfs://hdfsmaster:9000/alluxio/".
+# Check Alluxio documentation for more UnderFS storage options.
+#
+export ALLUXIO_UNDER_FS_DIR="${ALLUXIO_HOME}/underFSStorage"
+
+# ALLUXIO_DATA_CLEAR
+#
+# Alluxio has data files that are written to UnderFS and a journal file
+# containing metadata so the state of Alluxio filesystem is saved
+# and can be reused on multiple Alluxio runs.
+#
+# This flag enabled forces that data to be wiped which results
+# in a clean install of Alluxio every time Magpie runs it.
+#
+# Specify yes or no. Defaults to yes.
+export ALLUXIO_DATA_CLEAR="yes"
+
+# Memory capacity of each worker node
+#
+# By default, Alluxio allocates 2/3 of total system memory,
+# which will not be a valid setting for Magpie run when other
+# components are running on the same hosts as Alluxio,
+# so this variable was created.
+#
+export ALLUXIO_WORKER_MEMORY_SIZE=16GB
+
+# ALLUXIO_WORKER_TIER0_PATH
+#
+# Alluxio will use RAM File System as caching storage.
+# In this variable, there could be set path for tiered storage level 0.
+# Default settings are /mnt/ramdisk on Linux and /Volumes/ramdisk on OSX.
+#
+# To avoid sudo requirement but using tmpFS in Linux,
+# set here /dev/shm and ALLUXIO_RAMFS_MOUNT_OPTION to NoMount option.
+export ALLUXIO_WORKER_TIER0_PATH=/mnt/ramdisk
```

Comment:
i know i'm not the most consistent in Magpie, but lets do double quotes here, just for consistency to the configs below